### PR TITLE
MF-491  Patient registration button opens up Edit patient page

### PR DESCRIPTION
**Pull request by brandones**
https://issues.openmrs.org/browse/MF-491

This ensures that the `currentPatientUuid` is updated *prior* to new components mounting, and ensures that if `currentPatientUuid` is null, then `currentPatient` is treated as null as well.

### CODE REVIEW

File: packages/esm-error-handling/README.md [hunk 1]
```diff
@@ -21,3 +21,5 @@
 
 
 
+
```
Comment (FlorianRappl) [inline packages/esm-error-handling/README.md:24]: Hm there might be a bug in the document.sh - maybe you can check it.

Just noticed that 2 new empty lines are appended each time it runs (or so it seems).

**Review by FlorianRappl — COMMENTED** [review]
LGTM! :beers:

File: packages/esm-error-handling/README.md (hunk 1 above)
Comment (brandones) [reply packages/esm-error-handling/README.md]: Uh-oh, you're right. Looks like it's a bug in markdown-toc. I've added a mitigation to this PR.

Comment (FlorianRappl) [reply packages/esm-error-handling/README.md]: Awesome - you are the man! :beers:

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**